src/Twig/SonataAdminRuntime.php

```diff
@@ -42,7 +42,8 @@ public function __construct(Pool $pool)
     public function getUrlSafeIdentifier(object $model, ?AdminInterface $admin = null): ?string
     {
         if (null === $admin) {
-            $class = ClassUtils::getClass($model);
+            // NEXT_MAJOR: Change to `\get_class($model)`
+            $class = BCHelper::getClass($model);
```

Comment:
why in this case you dont use modelManager?

Reply:
I dont have one since I dont have any admin.

Also this is used in twig template as a twig extensions ; I m not sure were supposed to work on proxy object.
As a good example how do you know if u need to usr the resolver from orm or mongo